elixir/apps/domain/test/support/fixtures/policies_fixtures.ex

```diff
@@ -0,0 +1,84 @@
+defmodule Domain.PoliciesFixtures do
+  alias Domain.{AccountsFixtures, ActorsFixtures, AuthFixtures, ResourcesFixtures}
+
+  def policy_attrs(attrs \\ %{}) do
+    name = "policy-#{counter()}"
+
+    Enum.into(attrs, %{
+      name: name,
+      actor_group_id: nil,
+      resource_id: nil
+    })
+  end
+
+  def create_policy(attrs \\ %{}) do
+    attrs = policy_attrs(attrs)
+
+    {account, attrs} =
+      Map.pop_lazy(attrs, :account, fn ->
+        AccountsFixtures.create_account()
+      end)
+
+    {subject, attrs} =
+      Map.pop_lazy(attrs, :subject, fn ->
+        actor = ActorsFixtures.create_actor(type: :account_admin_user, account: account)
+        identity = AuthFixtures.create_identity(account: account, actor: actor)
+        AuthFixtures.create_subject(identity)
+      end)
+
+    {actor_group_id, attrs} =
```

Comment:
I would accept actor_group struct instead of just ID to keep fixtures consistent, in most places we expect structs.

Reply:
👍 I updated this to accept both an actor_group and/or an actor_group_id (same for resource)